ietf-optical-impairment-topology.tree

```diff
@@ -243,7 +248,12 @@ module: ietf-optical-impairment-topology
                 |  +--ro amplifier
                 |     +--ro type-variety    string
                 |     +--ro operational
-                |        +--ro amplifier-element* []
+                |        +--ro amplifier-element*
+                |                [amplifier-path-id stage-order]
+                |           +--ro amplifier-path-id
+                |           |       uint16
```

Comment:
Also here I would simplify the definition as simple path-id, the context (amplifier) is already defined by the container.

Reply:
Amplifier-path-id is a proposed name that is one of the pending issue to discuss , since we were not sure it would be the best name to indicate the parallel amplifier element within an amplifier used for different frequency ranges.
So I would discuss that in the pending issues .